### PR TITLE
Updates to Retinotopy for missing code

### PR DESCRIPTION
**Pull request by CameronTEllis**
Gaps were revealed in the README and the analysis code for the pipeline to preprocess Retinotopy data.

### CODE REVIEW

File: prototype/link/scripts/retinotopy/FSLtoFS.sh [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+featDir=analysis/secondlevel/registration.feat
+
+if [ ! -d ${featDir}/stats/ ]; then
+mkdir ${featDir}/stats/
+fi
+
+yes | cp $REGCONCAT_DIR/default/func2highres.nii.gz ${featDir}/stats/func2highres.nii.gz
+
+echo 'Replacing bad transformation matrices with good ones'
+yes | cp -f scripts/retinotopy/identitymatrix.mat $featDir/reg/example_func2initial_highres.mat
```
Comment (tristansyates) [inline prototype/link/scripts/retinotopy/FSLtoFS.sh:39]: This identity matrix is not included in the code release. Also what is meant by "good transformation matrices"?

File: prototype/link/scripts/retinotopy/FSLtoFS.sh [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+updatefeatreg $featDir/
+
+#Allows FSL and Freesurfer to speak the same language
+#Take a look at this web address to see what this does: http://www.fmrib.ox.ac.uk/fsl/freesurfer/index.html
+echo 'Running FSL-FS interoperability programs'
+if [[ ${SCHEDULER} == slurm ]]
+then
+	SubmitName=`sbatch scripts/retinotopy/run_reg-feat2anat.sh ${featDir} ${FreesurferFolder}`
```
Comment (tristansyates) [inline prototype/link/scripts/retinotopy/FSLtoFS.sh:47]: This script is also missing

File: prototype/link/scripts/retinotopy/FSLtoFS.sh [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+fi
+
+echo Submitted $SubmitName to make an anat folder
+
+#What is the name of the file
+JobID=`echo $SubmitName | awk '{print $NF}'`
+
+if [[ ${SCHEDULER} == slurm ]]
+then
+	SubmitName=`sbatch --dependency=afterok:${JobID} scripts/retinotopy/run_feat2surf.sh ${featDir}`
```
Comment (tristansyates) [inline prototype/link/scripts/retinotopy/FSLtoFS.sh:60]: This script also missing

File: prototype/link/scripts/retinotopy/README.md [hunk 4]
```diff
@@ -10,7 +10,11 @@ Use **sbatch ./scripts/retinotopy/supervisor_retinotopy.sh** to run the necessar
 This directory can be created with FreeSurfer or by adapting iBEAT. To use Freesurfer you can use `scripts/run_recon.sh` as a script for launching the job. If using iBEAT then follow the steps described in the `convert_iBEAT-FS.md` README. This freesurfer directory must be able to have SUMA created from it.
 
 
-3. Make the cuts in the inflated surface using tksurfer
+3. Make FSL and Freesurfer interoperable
```
Comment (tristansyates) [inline prototype/link/scripts/retinotopy/README.md:13]: All of the README additions make sense!

**Review by tristansyates — CHANGES_REQUESTED** [review]

File: prototype/link/scripts/retinotopy/README.md (hunk 4 above)
Comment (CameronTEllis) [reply prototype/link/scripts/retinotopy/README.md:13]: Great!

File: prototype/link/scripts/retinotopy/FSLtoFS.sh (hunk 2 above)
Comment (CameronTEllis) [reply prototype/link/scripts/retinotopy/FSLtoFS.sh:47]: Great catch, added

**Review by tristansyates — APPROVED** [review]

**Pull request merged**